src/root/Makefile.am

```diff
@@ -21,9 +23,14 @@ all:
 	mkdir -p $(srcdir)/static/js
 	unzip -u -d $(srcdir)/static $(BOOTSTRAP)
 	unzip -u -d $(srcdir)/static/js $(FLOT)
+	unzip -u -d $(srcdir)/static $(FONTAWESOME)
 
 install-data-local: $(ZIPS)
 	mkdir -p $(hydradir)/static/js
 	cp -prvd $(srcdir)/static/js/* $(hydradir)/static/js
 	mkdir -p $(hydradir)/static/bootstrap
-	cp -prvd $(srcdir)/static/bootstrap/* $(hydradir)/static/bootstrap
+	cp -prvd $(srcdir)/static/$(basename $(BOOTSTRAP))/* $(hydradir)/static/bootstrap
+	mkdir -p $(hydradir)/static/fontawesome/css
+	cp -prvd $(srcdir)/static/$(basename $(FONTAWESOME))/css/all.css $(hydradir)/static/fontawesome/css
+	mkdir -p $(hydradir)/static/fontawesome/webfonts
+	cp -prvd $(srcdir)/static/$(basename $(FONTAWESOME))/webfonts/* $(hydradir)/static/fontawesome/webfonts
```

Comment:
It looks like this doesn't actually work when in a dev shell:

```
[nix-shell:~/projects/github.com/NixOS/hydra/src/root]$ make install-data-local
mkdir -p /nix/store/sj3rallg7arl4bp009j0ydgdzhkls590-hydra-0.1.20210318.5c8e6d7/libexec/hydra/root/static/js
mkdir: cannot create directory ‘/nix/store/sj3rallg7arl4bp009j0ydgdzhkls590-hydra-0.1.20210318.5c8e6d7’: Read-only file system
make: *** [Makefile:529: install-data-local] Error 1
```

like this make target is only used for install. I think this probably needs to be run as part of the `all` step... not sure, though... experimenting ...

Reply:
From the output I would have guessed that this must always have failed as it looks to be the first line of the rule
```
	mkdir -p $(hydradir)/static/js
```
which is the same as it was before.

To be honest, I had always assumed none of the install targets were valid inside a dev shell unless you redirect`$out` to somewhere that can be written. Is this not the case?

Reply:
I'm assuming that  was the case, so I'm marking this as resolved. If not, please unmark. Thanks!

PS: Thanks very much for the review effort and especially the rebase work! Much appreciated.